ci-operator/config/shiftstack/ci/shiftstack-ci-release-4.17__techpreview.yaml

```diff
@@ -39,7 +39,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: e2e-openstack-ovn-parallel
+- as: e2e-openstack-ovn-techpreview
```

Comment:
nit: any reason to drop the `-parallel` suffix? Just making sure this isn't e.g. a rebase snafu.

Reply:
Yes, sorry I forgot to mention in the commit message. I removed the `-parallel` suffix to be consistent with the jobs from other platforms.